libs/langgraph/langgraph/types.py

```diff
@@ -101,7 +101,10 @@ def default_retry_on(exc: Exception) -> bool:
 
 
 class RetryPolicy(NamedTuple):
-    """Configuration for retrying nodes."""
+    """
+    !!! version-added "Added in v0.2.24"
+        The RetryPolicy class was introduced in version 0.2.24 (September 2024).
```

Comment:
This doesn't provide any additional information. We just want a minimal version added.

```suggestion
```

---

A description could be useful if we start using ..version-modified